bin/Packager/CollectionPackagerBase.py

```diff
@@ -256,6 +262,50 @@ def copyFiles(self, srcDir, destDir, dontStrip) -> bool:
                     utils.sign([entry_target])
         return True
 
+    def symbolsDir(self):
+        """return absolute path to the symbol directory in imageDir
+        """
+        return os.path.join(self.imageDir(), 'symbols')
+
+    def internalPostInstall(self):
+        if not super().internalPostInstall():
```

Comment:
We need to move this to a class in a higher level. Not everyone is inheriting CollectionPackagerBase.
Mybe just use PackagerBase?